core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksActivity.java

```diff
@@ -138,6 +143,15 @@ protected void onCreate(Bundle savedInstanceState) {
     setupBookmarksAdapter();
     recyclerView.setAdapter(bookmarksAdapter);
 
+    bookmarksSwitch.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
```

Comment:
this can be converted to a lambda, it should be grey in AS, alt enter fot the autofix

Reply:
@macgills done.